app/assets/javascripts/app/project/project_sidebar.js

```diff
@@ -7,7 +7,8 @@ var baseSidebar = {
 
   selectReward: function(event){
     var url = this.$(event.currentTarget).data('new_contribution_url');
-    this.navigate(url);
+    var acceptContributions = this.$(event.currentTarget).data('accept_contributions');
```

Comment:
you're right, switching between ruby and js made me a bit confused :smile:. Thanks !